.gitignore

```
@@ -1,3 +1,4 @@
 bin
 .vscode/
 .idea/
+vendor.tar.gz
```

I think pycache should  be ignored too, right?
```suggestion
vendor.tar.gz
.tito/custom/__pycache__
```